src/Home/index.tsx

```diff
@@ -1,10 +1,143 @@
-import { Center, Box } from 'native-base';
-import React from 'react';
+import React, { useState } from 'react';
+import { FontAwesome } from '@expo/vector-icons';
+import {
+  AddIcon,
+  Box,
+  Circle,
+  Divider,
+  HStack,
+  SearchIcon,
+  useToast,
+  VStack,
+} from 'native-base';
+import { ScrollView } from 'react-native';
+import { Tile } from './tile';
+import { ModalSelection } from './ModalSelection';
+import { API_URL } from '../constants';
+import { Game } from '../interface';
+
+export function Home({ navigation, player }: any) {
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [showModal, setShowModal] = useState<boolean>(false);
+  const toast = useToast();
+
+  const navigateToGameState = (id: number) => {
+    navigation.navigate('GameState', {
+      gameId: id,
+    });
+  };
+
+  const navigateToGameSelector = () => {
+    navigation.navigate('GameSelector');
+  };
+
+  const navigateToUserAccount = () => {
+    navigation.navigate('Account');
+  };
```

Comment:
It can be interesting in the future to have a harmonize router. It could be, for example, a `useNavigation` hook returning these methods.